src/main/java/HtmlUtil.java

```diff
@@ -5,46 +5,38 @@
 public class HtmlUtil {
 
     public static String testableHtml(PageData pageData, boolean includeSuiteSetup) throws Exception {
```

Comment:
the parameter name "includeSuiteSetup" might be a bit misleading since it also triggers the inclusion of the suite teardown in addition to the suite setup in the output.